src/pages/people-page/PeoplePage.tsx

```diff
@@ -0,0 +1,118 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { useParams } from 'react-router-dom';
+
+export const PeoplePage = () => {
+  const [peopleList, setPeopleList] = useState<Person[]>([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const { peopleSlug } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
Instead of using setIsLoading(true) here, you could make the initial state as _true_ and skip this setState action